go.mod

```diff
@@ -105,8 +105,7 @@ require (
 	golang.org/x/time v0.11.0
 	google.golang.org/genproto/googleapis/rpc v0.0.0-20250218202821-56aae31c358a
 	google.golang.org/grpc v1.72.2
-	google.golang.org/grpc/cmd/protoc-gen-go-grpc v1.5.1
-	google.golang.org/protobuf v1.36.6
+	google.golang.org/protobuf v1.36.9
```

Comment:
Is the bump to 1.36.9 related to these changes?

Reply:
The bump is related to the way I ran the command to install it. I did `go get -tool` and it decided to upgrade it. I figured a patch upgrade wasn't a big deal and was likely healthy for other reasons so I kept it.